profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/functions_to_wrap.c

```diff
@@ -58,13 +59,28 @@ enum FUNCTION_ID
 // counters: one byte per function
 __thread unsigned long long functions_entered_counter = 0;
 
+// This variable is used to indicate to the profiler that the application is crashing
+// and it should not collect samples while the app is crashing.
+// At crash time, the .NET runtime calls fork() to create a child process which will
+// be in charge of collecting a crash dump (by calling execve()) while the parent is waiting
+// for the child to finish.
+// By calling fork(), the child process and the parent process will have their own address space,
+// which means that the child process won't be able to modify the parent process's variables.
+// We need a way to enable communication between the child and parent processes.
+// This is done by creating a shared memory region and use it as a flag to indicate that
+// the application is crashing.
+// This variable will be a pointer to that shared memory region.
 __attribute__((visibility("hidden")))
-atomic_int is_app_crashing = 0;
+int* is_app_crashing = NULL;
```

Comment:
I would rename the variable to **p_is_app_crashing** to make it more explicit that it is a pointer